README.md

```diff
@@ -58,24 +58,27 @@ For more details, refer to [cluster_up_down.md][].
 
   - see [Fedora32.md][]
 
+- **Note about Docker Hub (docker.io image registry)**
+
+  - You *must* have a [Docker Hub] account of some kind for pulling necessary
+    images. Depending on how you use OSBS Box, you *might* require a paid account,
+    as you *might* hit the free account's hourly limit on image pulls.
+  - Consult [dockerhub.md][] for more details.
+
 ### Deployment steps
 
 1. If you haven't already, `git clone` this repository
-1. Take a look at [group_vars/all.yaml][]
+1. Take a look at [group_vars/all.yaml][][^1]
```

Comment:
This doesn't seem to work as expected: https://github.com/ben-alkov/osbs-box/tree/enable-use-dockerhub-secret#deployment-steps

Reply:
TIL GitHub *seems* to not care about [customer complaints](https://github.com/github/markup/issues/498) regarding basic writing tools (for 5+ years!)

Reply:
The workaround is ugly hacky raw HTML